Packages/com.unity.inputsystem/InputSystem/Actions/InputBindingResolver.cs

```diff
@@ -218,23 +218,38 @@ public unsafe void AddActionMap(InputActionMap map)
                             action = currentCompositeAction;
                         }
 
+                        // If it's a composite, start a chain.
```

Comment:
The changes starting here look pretty invasive but essentially only one simple thing changed: before, the binding state for a composite was only set up if the binding was not disabled. Also, binding states for composites were written out from a dedicated path which then earlied out of the current loop iteration.

Now, the binding state is *always* written at the end. For composites, for part bindings, and for "normal" bindings. Which now allows to safely disable any kind of binding.

This is now used if the action for a "normal" or composite binding cannot be find. If that is the case, the binding and all its part bindings are disabled and thus will not get their controls resolved.